cmd/docgen/handlers.go

```diff
@@ -219,7 +219,7 @@ func eventsForAction(action flows.Action) (json.RawMessage, error) {
 		}
 
 		// give all our example events a fixed created on time
-		event.SetCreatedOn(session.Environment().Now())
+		event.SetCreatedOn(utils.Now())
```

Comment:
Previously some times were mockable via the environment. Now utils.Now() provides system wide mocking and environment.Now() is just that time in the env timezone.